test/e2e/e2e_test.go

```diff
@@ -271,6 +271,35 @@ var _ = Describe("e2e control plane", func() {
 
 		framework.ExpectNoError(<-errChan)
 	})
+
+	ginkgo.It("should provide Internet connection continuously when all pods on master node are killed", func() {
+		ginkgo.By("Running container which tries to connect to 8.8.8.8 in a loop")
+
+		podChan, errChan := make(chan *v1.Pod), make(chan error)
+		go checkContinuousConnectivity(f, "", "connectivity-test-continuous", "8.8.8.8", 53, 30, podChan, errChan)
+
+		testPod := <-podChan
+		framework.Logf("Test pod running on %q", testPod.Spec.NodeName)
+
+		time.Sleep(5 * time.Second)
+
+		podClient := f.ClientSet.CoreV1().Pods("")
+
+		podList, _ := podClient.List(metav1.ListOptions{})
+		for _, pod := range podList.Items {
+			if pod.Spec.NodeName == "ovn-control-plane" && pod.Name != "connectivity-test-continuous" {
```

Comment:
skip etcd pod too, it runs in ram in the CI and won't recover, causing the cluster to break